BaseClasses.py

```diff
@@ -767,8 +767,25 @@ def item_count(self, item: str, player: int) -> int:
         Utils.deprecate("Use count instead.")
         return self.count(item, player)
 
+    def has_list(self, items: Iterable[str], player: int, count: int = 1) -> bool:
+        """Returns True if the cumulative count of items from a list present in state exceeds the specified number."""
```

Comment:
The way I would interpret it:
"exceeds" `>`
"meets or exceeds" `>=`

But that also makes it really wordy, and I'm trying to think of a way to make it less wordy...

Reply:
just "meets" would be fine, i think, because "too many" is not a thing for state.has.

Reply:
I think "meets" is also misleading, since people might assume it must be exactly equal to that value

Reply:
How about this as the doc string?
```
Has `player` collected at least `count` items matching `items`?
```

Reply:
I like that conceptually, but I'll reword it to not be a question